HOWTO_SNAP.md

```diff
@@ -0,0 +1,61 @@
+Steps:
+
+- First check snapcraft is currently building something fine:
```

Comment:
First check that snapcraft is able to build something:

Reply:
thanks